examples/README.md

```diff
@@ -0,0 +1,104 @@
+
+# Data Factory - Using the Testing Framework 
+
+To use the Testing Framework, is enough to import the published package resulting from the developement in the current repo located in src/data_factory_testing_framework.
```

Comment:
From a user perspective, should we always recommend to use the PyPI package as you also mention below?

Example:
_To use the Testing Framework, install the data-factory-testing-framework package from PyPI via your preferred package manager._